### DISCUSSION TITLE
[New Feature] Use House/Country ID instead of list index in map triggers and script actions

### DISCUSSION THREAD

**Discussion by TAK02** [Phobos-developers/Phobos#940] [Ideas and feedback]
### Description:
Allow map Trigger Actions and Script Actions to use both the ID and the list index for their parameters.

This request is specifically for everything with House# on ModEnc: https://modenc.renegadeprojects.com/ScriptTypes/ScriptActions & https://modenc.renegadeprojects.com/Actions_(maps)/RA2YR

But there's more; things like Theme#, Speech#, etc.

ScriptActions: 20 "Change House Ownership"
map Actions: 1 "Win", 2 "Lose", 3 "Production begins", ..., 14 "Change House", ..., 36 "All Change House", etc.

### Possible Implementation:
```ini
[Houses]
...
7=Confederation

[Actions]
01000018=1,36,0,7,0,0,0,0,A
```
parsable as
```ini
[Actions]
01000018=1,36,0,Confederation,0,0,0,0,A
```

**Comment by secsome** [comment]
I think it's not needed as map editor can automatically done the parse work, can you explain why?
